src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionFetchTestResultViewController.swift

```diff
@@ -0,0 +1,74 @@
+//
+// 🦠 Corona-Warn-App
+//
+//  ExposureSubmissionFetchTestResultViewController.swift
+
+import Foundation
+import UIKit
+
+class ExposureSubmissionFetchTestResultViewController: DynamicTableViewController {
+	
+	// MARK: - Init
+
+	init(_ viewModel: ExposureSubmissionFetchTestResultViewModel) {
+		self.viewModel = viewModel
+		super.init(nibName: nil, bundle: nil)
+	}
+
+	@available(*, unavailable)
+	required init?(coder: NSCoder) {
+		fatalError("init(coder:) has not been implemented")
+	}
+
+	// MARK: - Overrides
+
+	override func viewDidLoad() {
+		super.viewDidLoad()
+		setupView()
+	}
+
+	override var navigationItem: UINavigationItem {
+		navigationFooterItem
+	}
+
+	// MARK: - Public
+
+	enum CustomCellReuseIdentifiers: String, TableViewCellReuseIdentifiers {
```

Comment:
without public this is internal by default, please move to the correct section or mark public if needed

Reply:
ok, will be fixed